src/Compiler/Utilities/Activity.fs

```diff
@@ -4,10 +4,42 @@ namespace FSharp.Compiler.Diagnostics
 
 open System
 open System.Diagnostics
+open System.IO
+open System.Text
 
 [<RequireQualifiedAccess>]
 module Activity =
 
+    module Tags =
+        let fileName = "fileName"
+        let project = "project"
+        let qualifiedNameOfFile = "qualifiedNameOfFile"
+        let userOpName = "userOpName"
+        let length = "length"
+        let cache = "cache"
+        let cpuDelta = "cpuDelta(s)"
+        let realDelta = "realDelta(s)"
+        let gc0 = "gc0"
+        let gc1 = "gc1"
+        let gc2 = "gc2"
+        let outputDllFile = "outputDllFile"
```

Comment:
Thanks for doing all this. I'd expect constants to be PascalCase but that's not very important.

Reply:
There is an unrelated section about literals using the Literal attribute. Than PascalCase is expected.
But all the other let .. bindings of values I have seen around have been camelCase.

Reply:
Mhmhm yeah I probably mentally mixed it up with literals, nevermind then